src/msg/async/AsyncMessenger.cc

```diff
@@ -948,14 +948,14 @@ int AsyncMessenger::reap_dead()
 
   {
     std::lock_guard l2{deleted_lock};
+    num = deleted_conns.size();
```

Comment:
This is a narrowing conversion from unsigned to signed. Might as well fix the return type of this method to return `std::size_t`.